playbooks/hypervisor_deploy_subscriber_tests.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Are you sure for the name of the file ?
I would rather name it "tests_deploy_subsriber_tests.yaml"
subscribers can only go on hypervisor, so it is not needed in the name

Reply:
I named it in that way to differentiate subscriber tests run on VM and  subscriber tests run on hypervisor. 
But I think finally it would be simpler if all is run in one playbook.